server/aws-lsp-codewhisperer/src/language-server/agenticChat/tools/toolShared.ts

```diff
@@ -29,6 +32,31 @@ export async function validatePath(path: string, exists: (p: string) => Promise<
         throw new Error(`Path "${path}" does not exist or cannot be accessed.`)
     }
 }
+export async function fileExists(path: string, workspace: Workspace): Promise<boolean> {
```

Comment:
these functions seem generally relevant, not specific to agenticChat. should they live in lsp-core ?